fearless/Common/View/UIFactory.swift

```diff
@@ -210,6 +212,53 @@ final class UIFactory: UIFactoryProtocol {
         )
     }
 
+    func createAmountInputView(filled: Bool) -> AmountInputView {
+        let amountInputView = AmountInputView()
+
+        if !filled {
+            amountInputView.triangularedBackgroundView?.strokeColor = R.color.colorWhite()!
+            amountInputView.triangularedBackgroundView?.highlightedStrokeColor = R.color.colorWhite()!
+            amountInputView.triangularedBackgroundView?.strokeWidth = 1.0
+            amountInputView.triangularedBackgroundView?.fillColor = .clear
+            amountInputView.triangularedBackgroundView?.highlightedFillColor = .clear
+        } else {
+            amountInputView.triangularedBackgroundView?.strokeWidth = 0.0
+            amountInputView.triangularedBackgroundView?.fillColor = R.color.colorDarkGray()!
+            amountInputView.triangularedBackgroundView?.highlightedFillColor = R.color.colorDarkGray()!
+        }
+
+        amountInputView.titleLabel.textColor = R.color.colorLightGray()
+        amountInputView.titleLabel.font = .p2Paragraph
+        amountInputView.priceLabel.textColor = R.color.colorLightGray()
+        amountInputView.priceLabel.font = .p2Paragraph
+        amountInputView.symbolLabel.textColor = R.color.colorWhite()
+        amountInputView.symbolLabel.font = .h4Title
+        amountInputView.balanceLabel.textColor = R.color.colorLightGray()
+        amountInputView.balanceLabel.font = .p2Paragraph
+        amountInputView.textField.font = .h4Title
+        amountInputView.textField.tintColor = R.color.colorWhite()
+        amountInputView.verticalSpacing = 2.0
+        amountInputView.iconRadius = 12.0
+        amountInputView.contentInsets = UIEdgeInsets(
+            top: 8.0,
+            left: UIConstants.horizontalInset,
+            bottom: 8.0,
```

Comment:
I suggest to move all insets to local Constants as we do in other views

Reply:
So we are moving to code based layout and suggestion here is that we don't move constants to the header because it is used in one place with clear naming